_posts/2019-04-25-safer-and-cleaner-seas-stm.md

```diff
@@ -0,0 +1,126 @@
+author: arto
+excerpt: Air traffic has been digitalized long ago. Now, the seas are going same route as well. Future holds much safer and cleaner marine traffic for all of us.
+tags:
+- Environment
+- Automation
+- Digitalization
+- Project
+- Distributed
+- Java
+- XML
+---
+**Human errors can be expensive**. When you are making decisions, and acting on them while not fully concentrating, it leaves room for mistakes. A typical human error might be writing your address wrong when ordering an item. Or copying a single digit of bank account number wrong, from a handwritten note. Perhaps you are acting on too little information, or are in a hurry. Those mistakes happen every day to someone. In most cases you can recover from them through some extra work, no harm done.
+
+![Accidents at sea (source listed at end of the article)](/img/safer-and-cleaner-seas/marine_casualty_information.png)
+
+Mistakes that happen when operating a moving vehicle of any kind, such as a car, train, ship or an airplane, tend to be much more severe. These mistakes are not just about mild embarrassment or some monetary cost. **Any mistakes involving a moving vehicle bring much more costs to recover**. Sometimes they may result in environmental disasters, bodily harm, or even loss of life. Doing routine work that has to be repeated just right is a major reason for human errors.
+
+This is why any kind of professional transportation has multiple overlapping safety controls in place to prevent any accidents. Still, occasionally things manage to go wrong, and accidents do happen. Sometimes it's a matter of failing equipment or parts, but often it can also be attributed to mistakes in judgment, not being informed enough, or trusting bad information.
+
+### Optimization and Situational Awareness
+
+Some of the lesser mistakes do not have such severe outcomes, but they still cost time, money, and extra fuel resources. If we were able to optimize better and avoid even smaller mistakes, it would instantly reduce the operating costs as well as benefit the environment. **Use less fuel, save some money, help save the planet**. By optimizing things, we can also expect to avoid some frustration from people who are running the ships, as well as people from the port authorities' offices, pilots, ship owners, coast guard, and tugboats.
+
+Finally, how about just improving the feeling of safety and confidence, by improving situational awareness? This can be done by bringing the right information in the right format, at the right time, for people who are making the decisions. A baseline for having any information outside your ships own sensors might have been at some point, and still very much is radio traffic. Radio traffic itself is very prone to miscommunication, and things might get lost in translation. There are protocols again to cope with this, but it's still difficult to get the situational awareness just right every time. If only we had a better way...
```

Comment:
"safety and confidence, by improving..."

Ei tarvitse pilkkua.

Reply:
"This can be done by bringing the right information in the right format, at the right time, for people who are making the decisions."

Tämä on jotenkin hankalan näköinen lause. Ehkäpä:

"This can be done by making sure that the people making the decisions have the right information, in the right format and at the right time."